deployment/SSL.md

```diff
@@ -0,0 +1,23 @@
+# SSL
+
+Some Aerie service configurations require connecting to internal JPL services (e.g. authentication via CAM). For this to work you need to add the proper internal JPL certificates to some of the Aerie containers.
```

Comment:
While this information is really important to capture and communicate to users, should we be placing this information in the public repo/part of the public docs? I'm pondering our other options. Maybe a separate set of docs specific for JPL users which dovetails with the public docs. 🤔

Reply:
Yeah I agree. The deployment docs seemed like the most logical place for now.

Reply:
We could put them in a folder `deployment/internal` to separate the docs. I can't think of a simple way to do this though without keeping a completely separate internal document set. 
Also related to this, `cam` auth is the default for our deployment, which implies internal deployment. Maybe we switch the default to `none` and do keep an internal document set?

Reply:
We resolved to move specific deployment concerns to a JPL internal docs repo therefore keeping the main project here generic in application.